CMakeLists.txt

```diff
@@ -135,4 +139,4 @@ endif()
 
 include_directories(${CMAKE_BINARY_DIR}/generated) # Globally include the build/generated directory
 
-add_subdirectory(src)
+add_subdirectory(src)
```

Comment:
Your editor is probably removing newlines on save. We keep newlines at the end of files.